frontend/src/feature-flags/excludeFeatureRules.ts

```diff
@@ -0,0 +1,12 @@
+import {ExcludeFeatureRule} from './featureFlagsProvider'
+
+export const EXCLUDE_FEATURE_RULES: ExcludeFeatureRule[] = [
```

Comment:
suggestion: to simplify these checks and also improve the readability, you can keep a map of features for every version, and build the supported features with a reduce.
```
const FEATURES_PER_VERSION = {
  '3.1.0': ['multiuser_cluster'],
  '3.2.0': ['lustre_persistent2', ...],
  '3.3.0': ['feature_x', ...], 
}

const supportedFeatures = FEATURES_PER_VERSION.reduce((acc, currentFeatures, version) => {
  if (userVersion >= version) {
    return [...acc, ...currentFeatures];
  }
  return acc;
}, []);
```

Reply:
I like the idea, I discarded this approach as it's specific to versions while it could make sense that features get disabled according to some different logic in the future (e.g. something about the user)

But I agree it's more readable, therefore we can change it if we don't want to think about other types of flags for now